.eslintrc

```diff
@@ -14,7 +14,7 @@
   },
   "overrides": [
     {
-      "files": [ "stories.js", "test.js", "src/stories/index.js", "src/setupTests.js" ],
+      "files": [ "stories.js", "test.js", "src/stories/index.js", "src/setupTests.js", "src/components/createComponent.js" ],
```

Comment:
build scripts, imo, should live in /scripts rather than /src, or potentially in /src/scripts/

There should also be an npm script that runs this, e.g. `npm run create-component`